deeplenstronomy/check.py

```diff
@@ -400,8 +401,13 @@ def check_input_distributions(self):
                                     errs.append("DISTRIBUTIONS." + userdist + ".STEP must be a positive integer")
                         # if params are specified, they must be in a list
                         if 'PARAMS' in self.config["DISTRIBUTIONS"][userdist].keys():
-                            if not isinstance(self.config["DISTRIBUTIONS"][userdist]['PARAMS'], list):
+                            if not isinstance(self.config["DISTRIBUTIONS"][userdist]['PARAMS']:
                                 errs.append("DISTRIBUTIONS." + userdist + ".PARAMS must be a list")
+                            else:
+                                user_df = self.config.["DISTRIBUTIONS"][userdist]['FILENAME']
```

Comment:
I think this should be
`params = self.config.["DISTRIBUTIONS"][userdist]['PARAMS']`, otherwise the `user_df` variable has no use and `params` is never defined.

Reply:
Resolved by e310684